class-tgm-plugin-activation.php

```diff
@@ -1630,11 +1630,15 @@ public function get_tgmpa_url() {
 			static $url;
 
 			if ( ! isset( $url ) ) {
+				$parent = $this->parent_slug;
+				if ( strpos( $parent, '.php' ) === false ) {
```

Comment:
You'll just have to keep telling me that again and again... I just can't get used to it. Anyways, fixed & squashed.